NSSNamespace/Initial_Bangazon_Site
# Es productlistduplicatecountfixed

Details:

When a user navigates to the Order/Cart view, he/she now sees each product in his/her cart listed only once, with a number in parentheses beside that product indicates its quantity, rather than duplicate listings of the same product.

To test:

1. Git fetch
2. Checkout `es-PRODUCTLISTDUPLICATECOUNT`
3. Select a customer with duplicate items in the cart and navigate to the Order/Cart view
4. Insure that the product count beside each item is accurate
5. Select a payment type and submit order
6. Ensure app still routes to order confirmation page
